src/components/OptionPanel/index.js

```diff
@@ -0,0 +1,29 @@
+import React, { PureComponent } from 'react';
+import './styles.css';
+import { Card, CardBody, CardTitle, CardText, Button, ButtonGroup } from 'reactstrap';
+
+export default class OptionPanel extends PureComponent {
+  getButtons = () =>
+    this.props.options.map((option, i) => this.generateButton(option.label, option.value, option.color, i));
+
+  generateButton = (label, value, color, counter) => (
```

Comment:
This should actually be a seperate component. That'd fix the performance problem below too